### PR TITLE
Fix Oauth S2S IMS scope parameter and example variables

### PR DESCRIPTION
**Pull request by pratyushbanerjee**
## Description

Fix IMS scope parameter and variables in example IMS request for Oauth Server-to-Server

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/authentication/ServerToServerAuthentication/IMS.md [hunk 1]
```diff
@@ -15,15 +15,15 @@ Parameters can be sent in the body or as query parameters. Passing parameters in
 |`grant_type`|Yes|Value should always be `client_credentials`|
 |`client_id`|Yes|The Client ID obtained from the [Adobe Developer Console](/console)|
 |`client_secret`|Yes|The value of client secret obtained from the [Adobe Developer Console](/console)|
-|`scopes`|Yes|The list of comma separated scopes you are requesting. To see the scopes available to your credential, visit your project your the [Adobe Developer Console](/console)|
+|`scope`|Yes|The list of comma separated scopes you are requesting. To see the scopes available to your credential, visit your project your the [Adobe Developer Console](/console)|
```
Comment (cjelger) [inline src/pages/guides/authentication/ServerToServerAuthentication/IMS.md:18]: The end of the sentence should also be fixed : "visit your project your the ..."

File: src/pages/guides/authentication/ServerToServerAuthentication/IMS.md [hunk 2]
```diff
@@ -15,15 +15,15 @@ Parameters can be sent in the body or as query parameters. Passing parameters in
...
 
 
 ### Request for OAuth Server-to-Server credential
 
 ```curl
 curl -X POST 'https://ims-na1.adobelogin.com/ims/token/v3' \
   -H 'Content-Type: application/x-www-form-urlencoded' \
-  -d 'client_id={CLIENT_ID}&client_secret={CLIENT_SECRET}&grant_type=client_credentials&scope={SCOPES}'
+  -d 'client_id={client_id}&client_secret={client_secret}&grant_type=client_credentials&scope={scope}'
```
Comment (cjelger) [inline src/pages/guides/authentication/ServerToServerAuthentication/IMS.md:26]: I'd rather keep the upper-case placeholder names, I think it's somehow more "visible" that those are different that the query parameter itself.

Comment (pratyushbanerjee) [reply src/pages/guides/authentication/ServerToServerAuthentication/IMS.md:26]: Makes sense. Should we make the variables upper case in all the other examples?

Comment (cjelger) [reply src/pages/guides/authentication/ServerToServerAuthentication/IMS.md:26]: IMO I'd just leave them as it is, I just thought that having `client_id={client_id}` was a bit more confusing than `client_id={CLIENT_ID}` but it's really a minor thing.

**Review by cjelger — APPROVED** [review]

**Pull request merged**